applications/nrf_desktop/configuration/common/hid_report_desc.h

```diff
@@ -84,6 +84,35 @@ static const uint8_t output_reports[] = {
 	REPORT_ID_KEYBOARD_LEDS,
 };
 
+/* Internal definitions used to calculate size of the biggest supported HID input report. */
+#define _REPORT_BUFFER_SIZE_MOUSE \
+	(IS_ENABLED(CONFIG_DESKTOP_HID_REPORT_MOUSE_SUPPORT) ? (REPORT_SIZE_MOUSE) : (0))
+#define _REPORT_BUFFER_SIZE_KEYBOARD_KEYS \
+	(IS_ENABLED(CONFIG_DESKTOP_HID_REPORT_KEYBOARD_SUPPORT) ? (REPORT_SIZE_KEYBOARD_KEYS) : (0))
+#define _REPORT_BUFFER_SIZE_SYSTEM_CTRL								\
+	(IS_ENABLED(CONFIG_DESKTOP_HID_REPORT_SYSTEM_CTRL_SUPPORT) ?				\
+	 (REPORT_SIZE_SYSTEM_CTRL) : (0))
+#define _REPORT_BUFFER_SIZE_CONSUMER_CTRL							\
+	(IS_ENABLED(CONFIG_DESKTOP_HID_REPORT_CONSUMER_CTRL_SUPPORT) ?				\
+	 (REPORT_SIZE_CONSUMER_CTRL) : (0))
+#define _REPORT_BUFFER_SIZE_BOOT_MOUSE \
+	(IS_ENABLED(CONFIG_DESKTOP_HID_BOOT_INTERFACE_MOUSE) ? (REPORT_SIZE_MOUSE_BOOT) : (0))
+#define _REPORT_BUFFER_SIZE_BOOT_KEYBOARD \
+	(IS_ENABLED(CONFIG_DESKTOP_HID_BOOT_INTERFACE_KEYBOARD) ? (REPORT_SIZE_KEYBOARD_BOOT) : (0))
+#define _MAX6(a, b, c, d, e, f)	MAX(MAX(a, b), MAX(MAX(c, d), MAX(e, f)))
+
+/* Size of the biggest supported HID input report that is part of input reports map. */
```

Comment:
You can now drop this comment

Reply:
Can we keep it (as it's part of the API and for consistency with `REPORT_BUFFER_SIZE_OUTPUT_REPORT`)?

Reply:
I would like to also ensure that user is aware exactly what is the meaning of the defined value

Reply:
I gave you approve (second time) so I do not see this change as crucial. Do as you want. Just pointing out that IMO currently this comment is redundant. But do as you wish :)